README.md

```diff
@@ -80,6 +81,16 @@ empty_array_mt
 --------------
 **syntax:** `setmetatable({}, cjson.empty_array_mt)`
 
+**NOTE**: Use of this field is *discouraged* as of the `2.1.0.6` release. This
```

Comment:
This version number is a guess based on the current release versions, to be updated if necessary.